lib/syskit/cli/log_runtime_archive.rb

```diff
@@ -41,6 +54,36 @@ def process_root_folder
                 end
             end
 
+            # Manages folder available space
+            #
+            # The method will check if there is enough space to save more log files
+            # according to pre-established threshold.
+            #
+            # @param [integer] free_space_low_limit: required free space threshold, at
+            #   which the archiver starts deleting the oldest log files
+            # @param [integer] free_space_delete_until: post-deletion free space, at which
+            #   the archiver stops deleting the oldest log files
+            def ensure_free_space(free_space_low_limit: @free_space_low_limit,
+                free_space_delete_until: @free_space_delete_until)
+                stat = Sys::Filesystem.stat(@target_dir)
+                available_space = stat.bytes_free
+
+                return if available_space > free_space_low_limit
+
+                until available_space >= free_space_delete_until
+                    files = @target_dir.each_child.select(&:file?)
+                    break if files.empty?
+
+                    available_space += files.sort.first.size
+                    files.sort.first.unlink
+                end
+            end
+
+            # Returns the file size in bytes
+            def self.size_of_file(path)
+                File.stat(path).size
+            end
```

Comment:
You dont really use this here, do you have any plans of using it in the future?

Reply:
No. I created it in the first version, but @rayssinha-soares went another way that does not require this anymore. Need to be removed @rayssinha-soares